VERSION.minor

```diff
@@ -0,0 +1 @@
+b20210108
```

Comment:
This should be removed

Reply:
Good catch: removed, added to .gitignore to prevent this happening in future